### PR TITLE
Add VibrationDevice sample

### PR DESCRIPTION
**Pull request by msalehmsft**
Plus minor doc fixes

### CODE REVIEW

File: Samples/VibrationDevice/CS/MainPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        private VibrationDevice _vibrationDevice = null;
+        private SimpleHapticsControllerFeedback _buzzFeedback = null;
+
+        public MainPage()
+        {
+            this.InitializeComponent();
+            _coreDispatcher = CoreWindow.GetForCurrentThread().Dispatcher;
+        }
+
+        private async Task GetVbrationDevice()
```
Comment (paulmon) [inline Samples/VibrationDevice/CS/MainPage.xaml.cs]: nitpick:GetVibrationDevice (spelling)

Comment (msalehmsft) [reply Samples/VibrationDevice/CS/MainPage.xaml.cs]: Will fix

File: Samples/VibrationDevice/CS/MainPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                _vibrationDevice = await VibrationDevice.GetDefaultAsync();
+                var status = $"Vibration device {(_vibrationDevice == null ? "NOT" : _vibrationDevice.Id)} found. ";
+                Debug.WriteLine(status);
+                Status.Text = status;
+
+                if (_vibrationDevice != null)
+                {
+                    _buzzFeedback = FindFeedback();
+                    status = $"Buzz feedback {(_buzzFeedback == null ? "NOT" : "")} supported by this device.";
+                    Debug.WriteLine(status);
```
Comment (paulmon) [inline Samples/VibrationDevice/CS/MainPage.xaml.cs]: nitpick: is this needed if you are showing it in the UI?

File: Samples/VibrationDevice/CS/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Vibration Device Sample
+
+You can find the source code for this sample by downloading a zip of all of our samples [here](https://github.com/Microsoft/Windows-iotcore-samples/archive/master.zip).
+
+In this sample, we show how to find and use a vibrartion device to play vibration notifications.
```
Comment (paulmon) [inline Samples/VibrationDevice/CS/README.md]: nitpick: vibration (spelling)

File: Samples/VibrationDevice/CS/MainPage.xaml.cs (hunk 2 above)
Comment (msalehmsft) [reply Samples/VibrationDevice/CS/MainPage.xaml.cs]: This is unlikely to happen, as vibration devices need to support this feedback type.

File: Samples/VibrationDevice/CS/README.md (hunk 3 above)
Comment (msalehmsft) [reply Samples/VibrationDevice/CS/README.md]: Thx, will fix

**Pull request merged**